python/README.md

```diff
@@ -33,7 +33,6 @@ Ambassador uses several TCP ports while running. All but one of them are in the
 | 8003 | `entrypoint/ambex`                    | Internal `ambex` snapshot access; not exposed outside pod                                                          |
 | 8004 | `diagd`                               | Internal `diagd` access when `AMBASSADOR_FAST_RECONFIGURE` is set; not exposed outside pod                         |
 | 8005 | `entrypoint/external_snapshot_server` | Exposes configuration snapshots for integration with other tools, such as the Ambassador Agent                     |
-| 8006 | `entrypoint/agent`                    | Receives GRPC calls made by the Envoy Metrics Server                                                               |
 | 8080 | `envoy`                               | Default HTTP service port                                                                                          |
```

Comment:
Does this need to be removed or reworded if we are going to use 8080 for the metrics server?

Reply:
This is the listing of the ports in use on the main `emissary` Deployment; the agent doesn't run on that Deployment, it runs on its own `emissary-agent` deployment.